### DISCUSSION TITLE
[UI] UI for likes

### DISCUSSION THREAD

**Discussion by furkanmutlu** [MutluMinds/fullstack-image-app#61] [Q&A] [answered]
Since now we have the like functionality, It would be nice to have a page/tab to display users likes.

Questions:
>  Should we have separate like page for each API?  If yes, what are the possible UI solution comes to your mind ?
>  Should we have a one page with all likes ? Segmented by its API source? 
> Any other ideas ?

**Comment by elyessAR** [comment]
interesting...but I didn't get the whole idea, is this going to be to display the number of likes of each gif/picture?

**Reply by furkanmutlu** [reply]
hmmm.. thats another interesting idea. We can also think of displaying number of likes for further story. But this one is about displaying/storing users favourite gifs. So that user can come 1 month later and see his favourite ones.

**Reply by elyessAR** [reply]
oh so it's a client side thing, I get it now.

**Comment by furkanmutlu** [comment, marked as answer]
Yeah thats the question. It will be either for each page/api, with tabs like below: (just created on bootstrap editor)
<img width="841" alt="image" src="https://user-images.githubusercontent.com/45897337/209784897-b403d8b0-6734-4c1f-9820-4b51fa24d44b.png">


OR we will create a new page, with collapsable segments/apis. And all likes will be stored in there.

**Reply by elyessAR** [reply]
both solutions are convenient, I personally vote for the first one. any other opinions would be appreciated.

**Reply by furkanmutlu** [reply]
I feel the same. Or we could do both ? It wouldnt hurt 😄

**Reply by furkanmutlu** [reply]
but lets start for now with implementing first one only. I will create UI  designs today and issue according to it. Thanks for your opinion.
